remo/api.py

```diff
@@ -206,13 +206,17 @@ def list_datasets(self):
         url = self.url(backend.v1_datasets)
         return self.get(url).json()
 
-    def list_dataset_contents(self, dataset_id, **kwargs):
+    def list_dataset_contents(self, dataset_id, limit=None, **kwargs):
         # TODO: need to filter with annotation_id
         url = self.url(backend.v1_dataset_images.format(dataset_id), **kwargs)
```

Comment:
@melodiCyb I guess you can just add **limit** here, like

url = self.url(backend.v1_dataset_images.format(dataset_id), limit)